### ISSUE TITLE
Suggestion: Add seek function

### ISSUE THREAD

**Issue by HeemPlayz** [Androz2091/discord-player#23]
Good Evening

I really love this package, since it is too simple to make, I hope you add a seek function
Example: seek 1
So when I run this command, the player will seek the song to minute 1, and like this

another suggestion but not necessary but if it is easy to make, I hope you make it
the 2nd suggestion is to make a function to get the volume, instead of only adjusting the volume, you can see the current volume.

Thanks for reading

**Comment by Androz2091** [comment]
Hello
Yes, a `seek()` function is a good idea. And you can already get the current volume:
```js
const queue = client.player.getQueue('guildID');
console.log(queue.volume); // 100 (default)
```

**Comment by HeemPlayz** [comment]
Thanks a lot, I hope you added it soon.

**Comment by HeemPlayz** [comment]
And if you add spotify too, It will be awesome

**Comment by HeemPlayz** [comment]
I have a problem, when changing the volume, and displaying it, it shows the default (100), did I do anything wrong?

`const emotes = require ("../config/emojis.json");
const embedColor = "#36393e";
const embedSuccess = "#22BF41";
const embedFail = "#f30707";

module.exports.run = async (client, message, args) => {

    if(!message.member.voice.channel) return message.channel.send({embed: {color: embedFail, description: `You must be in a voice channel!` }})

    if(!client.player.isPlaying(message.guild.id)) return message.channel.send({embed: {color: embedFail, description: `Nothing is being played!` }})
    if(args[0]) {
    if(isNaN(args[0])) return message.channel.send({embed: {color: embedFail, description: `Please enter a valid number!` }})
   if (args > 150) return message.channel.send({embed: {color: embedFail, description: `Volume can be only between \`150\` - \`20\`` }})
  if (args < 20) return message.channel.send({embed: {color: embedFail, description: `Volume can be only between \`150\` - \`20\`` }})
  client.player.setVolume(message.guild.id, parseInt(args.join(" ")));

    message.channel.send({embed: {color: embedSuccess, description: `Volume set to \`${args.join(" ")}\` ` }})
} else if(!args[0]) {
    const queue = client.player.getQueue(message.guild.id);
    message.channel.send({embed: {color: embedSuccess, description: `Volume: \`${queue.volume}\` ` }})
}
};

module.exports.config = {
    name: "vol",
    aliases: ["volume"]
};`

**Comment by Androz2091** [comment]
Your code is correct but looks like there is a bug in the package. Will be fixed.

**Comment by HeemPlayz** [comment]
so when fixing it?

**Comment by Androz2091** [comment]
This is fixed on the develop branch. You can use it if you can't wait for the v2 to be released.

**Comment by HeemPlayz** [comment]
When the new official update will be released?

**Comment by Androz2091** [comment]
It should be fixed in v3.1.0. #134 will be used for the seek suggestion
